docs/source/markdown/podman-volume-rm.1.md

```diff
@@ -39,6 +39,15 @@ $ podman volume rm --all
 $ podman volume rm --force myvol
 ```
 
+## Exit Status
+  **0**   All specified volumes removed
+
+  **1**   One of the specified volumes did not exist, and no other failures
+
+  **2**   One of the specified volumes is being used by a container
+
+  **125** The command fails for a reason other than volume did not exist
```

Comment:
```suggestion
  **125** The command fails for a reason other than volume did not exist or network in use
```

Reply:
This is a volume, nothing to do with Network